.github/settings.yml

```
@@ -26,4 +26,15 @@ collaborators:
   
   - username: mkorbi
     permission: admin
+    
```

Just below the CNCF staff `admin` section, add a similar comment for users with `maintain` access. I suggest non-cncf staff for now have `maintain` access. See https://probot.github.io/apps/settings/ for all settings.

_Note I can not add that comment in the right place on this PR because it's not part of the current PR changes._

```suggestion
  # maintain: Bootstrap group volunteers prior to governance voting
```